bridge/contracts/nftbridge/NFTBridge.sol

```diff
@@ -399,14 +398,20 @@ contract NFTBridge is
     address tokenAddress,
     address to,
     uint256 tokenId
-  ) public override {
-    address sender = _msgSender();
+  ) public payable override {
     address tokenCreator = getTokenCreator(tokenAddress, tokenId);
 
     // Transfer the tokens on IERC20, they should be already Approved for the bridge Address to use them
-    IERC721(tokenAddress).safeTransferFrom(sender, address(this), tokenId);
+    IERC721(tokenAddress).safeTransferFrom(_msgSender(), address(this), tokenId);
 
     crossTokens(tokenAddress, to, tokenCreator, "", tokenId);
+
+    if (fixedFee > 0) {
+      require(msg.value >= fixedFee, "NFTBridge: value is smaller than fixed fee");
+
+      // Send the payment to the MultiSig of the Federation
+      federation.transfer(fixedFee);
```

Comment:
refund unused value (msg.value - fixedFee)  to the sender

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/248/commits/a74bb02b193df4106ec9a82bad058a89f839f2d7

Reply:
ty